lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -654,6 +654,11 @@ def create_module_map(native_target)
               update_changed_file(generator, path)
               add_file_to_support_group(path)
 
+              if path != (linked_path = target.module_map_path)
```

Comment:
can this be tested?

Reply:
integration specs cover this pretty thoroughly, I can look into adding a unit test